widget/embedded/src/components/HeaderButtons/HomeButtons.tsx

```diff
@@ -26,6 +28,12 @@ export function HomeButtons(props: HomeButtonsPropTypes) {
     onClickNotifications,
   } = props;
 
+  const { config } = useAppStore();
+
+  const disabledNotifications = !!config?.disabledFeatures?.includes(
+    DisabledFeatures.Notifications
+  );
```

Comment:
A helper function that verifies disabled features could be useful.
(please check other files)

Reply:
fixed.